virt/config.go

```diff
@@ -25,7 +25,7 @@ var (
 
 		"data_dir": hclspec.NewDefault(
 			hclspec.NewAttr("data_dir", "string", false),
-			hclspec.NewLiteral(`"/opt/virt"`),
+			hclspec.NewLiteral(`"/var/lib/virt"`),
```

Comment:
I updated the README to point to `/opt/virt` on Friday, so we should fix the README again too.